README.md

```diff
@@ -1,5 +1,9 @@
 [![Build Status](https://travis-ci.org/JGCRI/hector.svg?branch=master)](https://travis-ci.org/JGCRI/hector)
 [![Build status](https://ci.appveyor.com/api/projects/status/2jmhvq7n2ap75u1o/branch/master?svg=true)](https://ci.appveyor.com/project/rplzzz/hector/branch/master)
+![test-coverage](https://github.com/JGCRI/hector/workflows/test-coverage/badge.svg) 
+[![codecov](https://codecov.io/gh/JGCRI/hector/branch/master/graph/badge.svg?token=EGM0lXDxRv)](undefined) 
```

Comment:
The Codecov badge should link to your results like the following; currently you have the link posted as `undefined`:

```
[![codecov](https://codecov.io/gh/JGCRI/hector/branch/master/graph/badge.svg?token=EGM0lXDxRv)](https://codecov.io/gh/JGCRI/hector)
```

The badge will activate once you get this on the master because that's where you have it pointing to.  You current branch results are visible though here:  https://codecov.io/gh/JGCRI/hector/branch/krd_unit_test

Reply:
@crvernon I've replaced this line in the Read ME and it is causing the test coverage to fail, when this is merged will that error be resolved?